pkg/agent/controller/networkpolicy/packetin_test.go

```diff
@@ -24,6 +25,74 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+// mockLogger implements PolicyCustomLogger.
+type mockLogger struct {
+	sync.Mutex
+	logged chan string
+}
+
+func (l *mockLogger) Printf(format string, v ...interface{}) {
+	l.Lock()
+	defer l.Unlock()
+	l.logged <- format
+}
+
+func newAntreaPolicyLogger() *AntreaPolicyLogger {
+	return &AntreaPolicyLogger{
+		logDeduplication: logRecordDedupMap{logMap: make(map[string]*logDedupRecord)},
+	}
+}
+
```

Comment:
please make sure you have a test like this one:
assuming the timer value is 100ms: log 11 packets, one every 10ms, and make sure you end up with 2 different log entries

Reply:
Will do, thanks for the suggestion! 💭